rustls-upkid/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rustls-upkid"
```

Comment:
I think we want two crates:

- A `upki` crate that builds a binary (containing both mirroring and updating functionality)
- A library for use in stuff like rustls-platform-verifier
- Potentially a library to share code between the other two?